app/views/comments/_edit.html.erb

```diff
@@ -8,8 +8,8 @@
       #imagebar {width:100%;}
     </style>
 
-
-    <%= render :partial => "editor/toolbar" %>
+    <%# toolbar needs location & comment_id to make unique element IDs%>
```

Comment:
here you could use an HTML comment perhaps? `<!-- comment -->` just to simplify a little?